config/.env.sample

```diff
@@ -1,27 +1,29 @@
 # DOCKER_COMPOSE: any - Setting a value indicates running via `docker compose`
 DOCKER_COMPOSE=TRUE
 
+# DOCKER_KEEP_ALIVE_TIME: any - Keep container running after the app exits.
+# Useful for debugging.  Time value may use the following optional suffixes…
+# "s": seconds (the default), "m": minutes, "h": hours, or "d": days.
+DOCKER_KEEP_ALIVE_TIME=''
```

Comment:
is '' considered as the same as the the default value 's'?

Reply:
The suffix is optional.